### PR TITLE
Factorial feature with loop

### PR DESCRIPTION
**Pull request by rajashri570**

### CODE REVIEW

File: Python_prog/addition.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# program to take two number as input and print sum of it
```
Comment (anurag-kqi) [inline Python_prog/addition.py:1]: Always follow good coding practices, and keep space around the operator. It improves readability of the code.

File: Python_prog/addition.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+num1=int(input("Enter a number1: "))
+num2=(int(input("Enter number2: ")))
```
Comment (anurag-kqi) [inline Python_prog/addition.py:3]: Why extra braces?

File: Python_prog/addition.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+sum=num1 + num2
+print(f"{num1}+{num2}={sum}")
```
Comment (anurag-kqi) [inline Python_prog/addition.py:5]: use spaces properly to format output properly

File: Python_prog/fact.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#WAP to print factorial of number 
+num=5
+fact=1
+for i in range(1,num+1):
```
Comment (anurag-kqi) [inline Python_prog/fact.py:4]: proper spaces

**Review by anurag-kqi — COMMENTED** [review]
Overall code looks good. Use proper spacing.

Please write recursive variants for these programs.

Also, can you find square root without using math.sqrt function?

File: Python_prog/addition.py (hunk 2 above)
Comment (rajashri570) [reply Python_prog/addition.py:3]: sorry sir,might i have added it by mistake after running

File: Python_prog/addition.py (hunk 3 above)
Comment (rajashri570) [reply Python_prog/addition.py:5]: sure sir

**Pull request merged**